### ISSUE TITLE
tailscaled doesn't start automatically

### ISSUE THREAD

**Issue by libjared** [tailscale-dev/deck-tailscale#16]
tailscaled.service lives in the system extension. The unit is WantedBy=multi-user.target, and is `enabled`.

On reboot, even though the system extension gets merged automatically (by systemd-sysext.service probably?), I still have to `sudo systemctl start tailscaled.service` manually every time.

**Comment by legowerewolf** [comment]
What installation method did you follow? Are you still based off the original blog post?

`tailscaled.service` should not live in the system extension. It should be in `/etc/systemd/system`. There's an order to how the system starts up and by the time the extension is loaded, systemd has already tried and failed to find the service file.

If you want to continue using the system extension method, clone the repo to your deck, checkout the branch `as-system-extension`, and run the `tailscale.sh` install script. 

There's a new installation method on the main branch that avoids system extensions. If you want to switch to using that, clone the repo and run the `tailscale.sh` script on the `main` branch.

**Comment by libjared** [comment]
You're right, I _was_ still based off the original blog post. Sorry, must have misremembered.

I saw some logic in that script that would handle cleaning up legacy installs, so I followed the readme.

I ran into the "flag -port: can't be the empty string" issue, so I deleted it manually then reinstalled. It said it wanted to make a backup of the override (even though the contents were the same) so I deleted the backup manually.

Paper cut bugs aside, it seems to be working! Thank you.
